package.json

```diff
@@ -11,13 +11,15 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.0.0-beta.0",
-    "react-scripts": "4.0.3"
+    "react-scripts": "4.0.3",
+    "typed.js": "^2.0.12"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint \"**/*.js\""
```

Comment:
Oops, thought we already had a linter since this project was generated from Matt's template repo, but I guess not. Either way, good call adding it!